check50/__main__.py

```diff
@@ -266,8 +266,14 @@ def main():
                         metavar="FILE",
                         help=_("file to write output to"))
     parser.add_argument("-v", "--verbose",
-                        action="store_true",
-                        help=_("display the full tracebacks of any errors (also implies --log)"))
+                        action="store",
+                        nargs="?",
+                        default="none",
+                        const="info",
+                        choices=["info", "debug"],
+                        help=_("sets the verbosity level and implies --log."
+                               " \"info\" displays the full tracebacks of errors and shows all commands run."
+                               " \"debug\" adds the output of all command run."))
```

Comment:
I would use single quotes here for the outer quotes instead of escaping double quotes in double quotes.

Reply:
Done!